cmake/simdjson-config.cmake.in

```diff
@@ -1,9 +1,6 @@
-@PACKAGE_INIT@
-
 include(CMakeFindDependencyMacro)
-if(@SIMDJSON_ENABLE_THREADS@)
-    set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
```

Comment:
Why isn't this needed anymore?

Reply:
I'm not sure what good is there in forcing it. I could use some explanation behind the logic here.

Reply:
@lemire knows a lot more than I do :)

Reply:
I do not think it matters much. The `CMAKE_THREAD_PREFER_PTHREAD` parameter is a workaround to avoid problems on some systems, but this can be fixed by the library user.